src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_conversion_emitters.cpp

```diff
@@ -0,0 +1,174 @@
+namespace aarch64 {
+
+template <cpu_isa_t isa>
+static void jit_convert_process(dnnl::impl::cpu::aarch64::jit_generator* h,
+                                const std::vector<size_t> &in_idxs, const std::vector<size_t> &out_idxs,
+                                ov::element::Type input_type, ov::element::Type output_type, bool is_saturated) {
+    if (input_type == output_type) {
+        using TReg = typename dnnl::impl::cpu::aarch64::cpu_isa_traits<isa>::TReg;
+        h->mov(TReg(out_idxs[0]).b16, TReg(in_idxs[0]).b16);
+        return;
+    }
+
+    switch (output_type) {
+        case ov::element::f32:
+            switch (input_type) {
+                case ov::element::f32:
+                    break;
+                case ov::element::i32:
+                    cvt_i32_to_f32<isa>(h, in_idxs, out_idxs);
+                    break;
+                case ov::element::f16:
+                    cvt_f16_to_f32<isa>(h, in_idxs, out_idxs);
+                    break;
+                case ov::element::i8:
+                case ov::element::u8:
+                    cvt_byte_to_i32<isa>(h, in_idxs, out_idxs, input_type.is_signed());
+                    cvt_i32_to_f32<isa>(h, out_idxs, out_idxs);
+                    break;
+                default:
+                    OV_CPU_JIT_EMITTER_THROW("Unsupported input type: ", input_type.get_type_name());
+            }
+            break;
+        case ov::element::i32:
+            switch (input_type) {
+                case ov::element::f32:
+                    cvt_f32_to_i32<isa>(h, in_idxs, out_idxs);
+                    break;
+                case ov::element::i32:
+                    break;
+                case ov::element::f16:
+                    cvt_f16_to_f32<isa>(h, in_idxs, out_idxs);
+                    cvt_f32_to_i32<isa>(h, out_idxs, out_idxs);
+                    break;
+                case ov::element::i8:
+                case ov::element::u8:
+                    cvt_byte_to_i32<isa>(h, in_idxs, out_idxs, input_type.is_signed());
+                    break;
+                default:
+                    OV_CPU_JIT_EMITTER_THROW("Unsupported input type: ", input_type.get_type_name());
+            }
+            break;
+        case ov::element::f16:
+            switch (input_type) {
+                case ov::element::f32:
+                    cvt_f32_to_f16<isa>(h, in_idxs, out_idxs);
+                    break;
+                case ov::element::i32:
+                    cvt_i32_to_f32<isa>(h, in_idxs, out_idxs);
+                    cvt_f32_to_f16<isa>(h, out_idxs, out_idxs);
+                    break;
+                case ov::element::f16:
+                    break;
+                case ov::element::i8:
+                case ov::element::u8:
+                    cvt_byte_to_i32<isa>(h, in_idxs, out_idxs, input_type.is_signed());
+                    cvt_i32_to_f32<isa>(h, out_idxs, out_idxs);
+                    cvt_f32_to_f16<isa>(h, out_idxs, out_idxs);
```

Comment:
**Suggestion**. Is it `i8`/`u8` => `f16` correct? Can we simplify here to just two instructions only? Note, please, source code is not tested. If yes, what about other similar cases?

For signed int8:
```
sshll(out.h8, in.b8, 0);  // signed shift left long
scvtf(out.h, out.h);      // signed integer convert to half precision floating-point
```
For unsigned int8:
```
ushll(out.h8, in.b8, 0);   // unsigned shift left long
ucvtf(out.h, out.h);       // unsigned integer convert to half precision floating-point
```

Reply:
Good point! Applied. Thanks Edward!

Reply:
Hi Edward! I find that single conversion instruction between f16 and i16 is only available for archtecture ARMv8.2-A or later versions. As the cpu isa asimd we supported do not distinguish ARMv8 with ARMv8.2-A, I still use three instructions to convert between f16 and i16.

I met crash on conversion SLT on ci machine ie-tests-linux-ubuntu20_arm64-cpu. With the help of validation team, I find that this is a Raspberry (Model name Cortex-A72) with archtecture ARMv8 that do not support single conversion instruction between f16 and i16. So I still use three instructions to make it compatible with ARMv8 platforms.

Please feel free to have further discussions. Thanks! @eshoguli cc' @dmitry-gorokhov